node/subsystem-util/src/lib.rs

```diff
@@ -65,8 +65,6 @@ pub mod reexports {
 	pub use polkadot_overseer::gen::{SpawnedSubsystem, Spawner, Subsystem, SubsystemContext};
 }
 
-/// A rolling session window cache.
-pub mod rolling_session_window;
```

Comment:
RIP RSW 😄

Reply:
Don't put more salt in the wound. I already feel like destroying cultural heritage.